OpenAdaptAI/OpenAdapt
# Add onboarding for new users

**What kind of change does this PR introduce?**

This PR introduces an onboarding mechanism to the dashboard.



**Summary**
This PR introduces a change where the dashboard is launched by default when the app is launched. The dashboard also displays an onboarding modal, which the user can dismiss and it won't pop up again. The modal contains an input box to capture the email of the user if they wan't to receive regular updates about OpenAdapt. It also contains a link that can be used to schedule a short meeting with @abrichr 


This PR address #711 

**Checklist**

**How can your code be run and tested?**
Run the app as usual `python -m openadapt.entrypoint`. The dashboard should launch by default, and you should see an onboarding form. Closing the modal (by clicking on X or outside the modal) or set a config variable to false, and the modal will not be shown again on subsequent page visits.



**Other information**
